peer/network.go

```diff
@@ -333,6 +336,11 @@ func (n *network) AppRequest(ctx context.Context, nodeID ids.NodeID, requestID u
 
 	log.Debug("received AppRequest from node", "nodeID", nodeID, "requestID", requestID, "requestLen", len(request))
 
+	if err := n.router.AppRequest(ctx, nodeID, requestID, deadline, request); err == nil {
+		return nil
+	}
+	// if we errored, it's possible that this is a non-sdk request
```

Comment:
Yeah it's a good question. I think broadly speaking, it might just be good to have metrics on the sdk. We can maybe have something when registering app protocols we register a metric namespace as well.